pkg/syncproto/sync_proto.go

```diff
@@ -214,6 +214,9 @@ type MsgServerHello struct {
 	// SyncerType the active syncer type; if not specified, implies that the server is an older Typha instance that
 	// only supports SyncerTypeFelix.
 	SyncerType SyncerType
+
+	// SupportsNodeResourceUpdates specifies whether this Typha supports node resource updates.
+	SupportsNodeResourceUpdates bool
```

Comment:
Perhaps `Provides...` instead of `Supports...`, to give a helpful hint about the direction of the information flow?